noop.go

```diff
@@ -2,8 +2,17 @@ package opentracing
 
 import "github.com/opentracing/opentracing-go/log"
 
-// A NoopTracer is a trivial implementation of Tracer for which all operations
-// are no-ops.
+// A NoopTracer is a trivial, minimum overhead implementation of Tracer
+// for which all operations are no-ops.
+//
+// The primary use of this implementation is in libraries, such as RPC
+// frameworks, that make tracing an optional feature controlled by the
+// end user. A no-op implementation allows said libraries to use it
+// as the default Tracer and to write instrumentation that does
+// not need to keep checking if the tracer instance is nil.
+//
+// For the same reason, the NoopTracer is the default "global" tracer
+// (see GlobalTracer and SetGlobalTracer functions).
```

Comment:
Ok. It is perhaps too strong to claim that NoopTracer is suitable for RPC, since it would need to bring its own baggage propagation to the table like TChannel, just as it is too strong to claim that it is not suitable for RPC.